src/js/components/message.jsx

```diff
@@ -35,16 +36,54 @@ export class MessageComponent extends Component {
         }
 
         const style = {};
-        if (isAppUser && this.props.accentColor) {
+
+        if (!this.props.mediaUrl) {
+          style.borderTopLeftRadius = style.borderTopRightRadius = style.borderBottomLeftRadius = style.borderBottomRightRadius = 16;
```

Comment:
Use classes instead, make an array of them and use `join(' ')` when setting the className on the element. I suggest you create really small classes that targets only one thing and then compose the style with that.